official/vision/beta/projects/yolo/utils/preprocessing_ops.py

```diff
@@ -0,0 +1,177 @@
+import tensorflow as tf
+import tensorflow.keras.backend as K
```

Comment:
You don't have to use tensorflow.keras.backend. Can you just use tf.keras ?

Reply:
Addressed